tests/test_setid.cpp

```diff
@@ -1,6 +1,4 @@
-//
-// test_setid.cpp
-//
+// Test of ENsetid EPANET API Function
```

Comment:
I suggest using the format `EN_****` etc. in order to be consistent with other test files.

Reply:
@Mariosmsk I revised the test's code as you suggested.